src/RedisOutputCachingMiddleware/RedisOutputCachingMiddleware.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Rather than a single target framework I think we want frameworks plural like: 
`<TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>`